apps/condo/domains/miniapp/schema/B2BAppAccessRight.test.js

```diff
@@ -239,6 +239,31 @@ describe('B2BAppAccessRight', () => {
                 })
             }, ERRORS.ACCESS_RIGHT_SET_NOT_FOR_CONNECTED_B2B_APP)
         })
+
+        test('Can not create or update "B2BAppAccessRight" if "accessRightSet"."type" != "GLOBAL"', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [serviceUser] = await registerNewServiceUserByTestClient(admin)
```

Comment:
Please make tests more real, create service user from support. 

Because its additional real-world protection, if you messed up with global logic this can be additional indicator, that existing business process isn't working